app/components/candidate_interface/new_references_review_component.rb

```diff
@@ -151,5 +183,13 @@ def feedback_status_label(reference)
         colour: t("candidate_reference_colours.#{reference.feedback_status}"),
       )
     end
+
+    def return_to_params
+      if @return_to_application_review
+        { 'return_to' => 'application-review' }
+      else
+        { 'return_to' => 'review' }
```

Comment:
i note `:review` has changed to `'review'` here; i guess rails isn't bothered?

Reply:
apparently not!